analyzer/src/main/kotlin/managers/Npm.kt

```diff
@@ -17,6 +17,8 @@
  * License-Filename: LICENSE
  */
 
+@file:Suppress("TooManyFunctions")
```

Comment:
Commit message: Wasn't the assumption previously that the parent module is either at '../..'  or at '../../..' for modules with namespaces? What replaces this assumption?

Reply:
1. Yeah, I didn't want to go into that level of detail. Still want me to change  ?
2. The assumption is eliminated by 'ancestorModuleDirs' parameter.

Reply:
In the current state the commit messages raises more questions than it answers to me. Why is that assumption removed? What was the problem? What is it replaced with? I would say either explain that (preferrably) or do not go into details at all.

Reply:
Done.